chart/compass/charts/kyma-environment-broker/values.yaml

```diff
@@ -83,6 +83,16 @@ lms:
   samlTenant: "TBD"
   disabled: false
 
+edp:
+  authURL: "TBD"
+  adminURL: "TBD"
+  namespace: "kyma-dev"
+  environment: "dev"
+  required: false
+  disabled: false
```

Comment:
I wonder if we should have this set to `true` by default, because we use overrides with other necessary values anyway